scripts/start.sh

```diff
@@ -108,6 +109,9 @@ echo Starting GAE development server
 # accidentally!
 
 if [[ "$MINIFICATION" == "true" ]]; then
+  # To start server with minified/optimized resources as in production environment.
+  # This will set MINIFICATION environment variable to true while preserving value of
+  # DEV_MODE variable.
```

Comment:
I think `while preserving value of DEV_MODE variable.` sounds confusing. Do we need to mention it?